src/pages/auth/components/LogInForm.jsx

```diff
@@ -71,13 +72,13 @@ export const LogInForm = () => {
             style={{ marginTop: '8px' }}
             type='submit'
             fullWidth
-            rightIcon={<LoginIcon />}
-            title={t('Log In')}
+            rightIcon={isLoading ? <Spinner /> : <LoginIcon />}
```

Comment:
the same, here you will be able to use jut loading={isLoading} property

Reply:
i add to button component new props "isLoading" and spinner